kenware/Assessment
# #162187176 add pagination to all get endpoints

#### What does this PR do?
- Add pagination to get endpoints

#### Description
- extend query_parser to include pagination
- refactored some code to reduce complexity
- test get jwt endpoint
- add test
#### How should this be manually tested?
- Create a local copy `git ft-add-pagination-class`
- Checkout to the branch `ft-add-pagination-class`
- Start the application with the command: `python manage.py runserver`
- On postman make a GET request to `{{base-url}}/api/v1/answers/`
- An authenticated user should be able to see list of answer objects with its pagination
- To run tests use `python manage.py`
#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#162187176](https://www.pivotaltracker.com/story/show/162187176)
